tensortrade/orders/broker.py

```diff
@@ -65,39 +53,43 @@ def cancel(self, order: Order):
             raise Warning(
                 'Cannot cancel order {} - order has already been cancelled.'.format(order.id))
 
+        """
         if order.status != OrderStatus.PENDING:
             raise Warning(
                 'Cannot cancel order {} - order has already been executed.'.format(order.id))
+        """
 
-        self._unexecuted.remove(order)
+        if order in self._unexecuted:
+            self._unexecuted.remove(order)
 
         order.cancel()
 
     def update(self):
-        for order, exchange in product(self._unexecuted, self._exchanges):
-            is_executable = order.is_executable_on(exchange) and self.clock.step >= order.start
 
-            if order in self._unexecuted and is_executable:
+        for order in self._unexecuted:
+            """
```

Comment:
Why is this commented?

Reply:
That can be uncommented. Also uncomment the code in the `cancel` method above.